hoot-core/src/main/cpp/hoot/core/algorithms/FindNodesInWayFactory.cpp

```diff
@@ -63,7 +63,7 @@ NodePtr FindNodesInWayFactory::createNode(const OsmMapPtr& map,
       {
         // Logging this as a warning, since it happens every once in awhile and life seems to go
         // on...
-        LOG_WARN("" << "Internal Error: Two nodes were found with the same coordinate.");
+        LOG_WARN("" << "Internal Problem: Two nodes were found with the same coordinate.");
```

Comment:
Not relevant to this PR but should we drop the duplicate one?

Reply:
That's a good question (made #2850).  I assume that there is somewhere in the code that is rejecting duplicate nodes, but if there is, I don't know where it is....worth checking.   I just the text to "Problem", b/c every time I would search through Jenkins regression test output for "error", it would pop up even though its not really an error at this point.  I think the original intent of the message is to tell you that something is wrong with one of the algorithms